indico/modules/attachments/util.py

```diff
@@ -104,8 +104,6 @@ def can_manage_attachments(obj, user):
         return True
     if isinstance(obj, db.m.Event) and obj.can_manage(user, 'submit'):
         return True
-    if isinstance(obj, db.m.Session) and obj.can_manage(user, 'coordinate'):
```

Comment:
Why would the coordinator be able to add session-wide (manager-only actions) attachments?